### DISCUSSION TITLE
Translation of quotes into other languages

### DISCUSSION THREAD

**Discussion by alonmar** [guedesfelipe/pls-cli#26] [Ideas]
Hello

Do you currently accept PR? I can help to be able to choose the language of the quotes
(maybe it will help in practicing other languages, who knows 🤷). 
I can add this new feature and start translating in Spanish

Thanks for this nice tool 😊

**Comment by guedesfelipe** [comment]
Hi @alonmar !!

Sure!! We are open to new PR's!

Feel free to contribute with the project. I'll make a contribution guide soon, but for now you can use `make install-dev` to develop.

Be welcome! If you have any problems you can ask me. 😃
